modules/addons/aws/external-dns/external-dns_aws_cleanup.sh.tmpl

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
```

Comment:
Should we `set -e` to make sure this returns early with a non-zero code if something goes wrong?